test/snippet/alignment/cigar_conversion/cigar_from_alignment.cpp

```diff
@@ -0,0 +1,26 @@
+#include <seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp>
+#include <seqan3/alignment/cigar_conversion/cigar_from_alignment.hpp>
+#include <seqan3/alignment/decorator/gap_decorator.hpp>
+#include <seqan3/alphabet/gap/gapped.hpp>
+#include <seqan3/alphabet/nucleotide/dna5.hpp>
+#include <seqan3/core/debug_stream.hpp>
+#include <seqan3/utility/views/slice.hpp>
+
+int main()
+{
+    using namespace seqan3::literals;
+
+    seqan3::dna5_vector reference = "ATGGCGTAGAGCTTCCCCCCCCCCCCCCCCC"_dna5;
+    seqan3::dna5_vector read = "ATGCCCCGTTGCTT"_dna5; // length 14
+
+    // we align the full query against the first 14 bases in the reference
+    seqan3::gap_decorator aligned_reference{reference | seqan3::views::slice(0, 14)};
+    seqan3::gap_decorator aligned_read{read};
+    // insert gaps (as in the alignment above)
```

Comment:
```suggestion
    // Insert gaps to represent the alignment:
    // ATGG--CGTAGAGC
    // |||X  |||X|  |
    // ATGCCCCGTTG--C
```

I copy/pasted the alignment mentioned from where the snippet is included. But it doesn't quite fit?